cli/flags.rs

```diff
@@ -1,4 +1,5 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+use crate::resolve_hosts::resolve_hosts;
```

Comment:
This method should be used in clap's `.validator()` method - that way error can be returned instead of panicking